client/src/components/rounds/Apply.tsx

```diff
@@ -74,6 +92,7 @@ function Apply() {
   }, [roundId]);
 
   if (props.roundStatus === RoundStatus.Error) {
+    dispatch(addAlert("error", applicationErrorTitle, applicationErrorBody));
```

Comment:
is this defined in any design? I like the idea of having the alert, but do we have an empty html body or do we redirect somewhere?

Reply:
I was thinking about that earlier... I was just using a generic message in the alert and html body for now and does not redirect. @gravityblast

Reply:
also had it in the wrong spot 🤦‍♂️